CHANGELOG.md

```diff
@@ -2,10 +2,15 @@
 
 ### Enhancements
 
+* **Add overlap option for split chunks.** Users can supply an `overlap: int` value on a partition function to start continuation chunks the specified number of characters "early", such that the last n characters of the prior chunk also appear as the first n characters of the following chunk, to improve semantic context.
+
 ### Features
 
 ### Fixes
 
+* **Fix out-of-order sequencing of split chunks.
+* **Fix breaking chunks on regex-metadata matches.
```

Comment:
```suggestion
* **Fix breaking chunks on regex-metadata matches.**
```
Please say more. what does this mean, why does it matter?

Reply:
k, fixed